runtime/doc/help.txt

```diff
@@ -38,169 +38,126 @@ Get specific help:  It is possible to go directly to whatever you want help
 Vim stands for Vi IMproved.  Most of Vim was made by Bram Moolenaar, but only
 through the help of many others.  See |credits|.
 ------------------------------------------------------------------------------
-						*doc-file-list* *Q_ct*
-BASIC:
-|quickref|	Overview of the most common commands you will use
-|tutor|		30-minute interactive course for beginners
-|copying|	About copyrights
-|iccf|		Helping poor children in Uganda
-|sponsor|	Sponsor Vim development, become a registered Vim user
-|www|		Vim on the World Wide Web
-|bugs|		Where to send bug reports
-
-USER MANUAL: These files explain how to accomplish an editing task.
-
-|usr_toc.txt|	Table Of Contents
-
-Getting Started ~
-|usr_01.txt|  About the manuals
-|usr_02.txt|  The first steps in Vim
-|usr_03.txt|  Moving around
-|usr_04.txt|  Making small changes
-|usr_05.txt|  Set your settings
-|usr_06.txt|  Using syntax highlighting
-|usr_07.txt|  Editing more than one file
-|usr_08.txt|  Splitting windows
-|usr_09.txt|  Using the GUI
-|usr_10.txt|  Making big changes
-|usr_11.txt|  Recovering from a crash
-|usr_12.txt|  Clever tricks
-
-Editing Effectively ~
-|usr_20.txt|  Typing command-line commands quickly
-|usr_21.txt|  Go away and come back
-|usr_22.txt|  Finding the file to edit
-|usr_23.txt|  Editing other files
-|usr_24.txt|  Inserting quickly
-|usr_25.txt|  Editing formatted text
-|usr_26.txt|  Repeating
-|usr_27.txt|  Search commands and patterns
-|usr_28.txt|  Folding
-|usr_29.txt|  Moving through programs
-|usr_30.txt|  Editing programs
-|usr_31.txt|  Exploiting the GUI
-|usr_32.txt|  The undo tree
-
-Tuning Vim ~
-|usr_40.txt|  Make new commands
-|usr_41.txt|  Write a Vim script
-|usr_42.txt|  Add new menus
-|usr_43.txt|  Using filetypes
-|usr_44.txt|  Your own syntax highlighted
-|usr_45.txt|  Select your language
-
-
-REFERENCE MANUAL: These files explain every detail of Vim.	*reference_toc*
-
-General subjects ~
-|intro.txt|	general introduction to Vim; notation used in help files
-|nvim.txt|	Transitioning from Vim
-|help.txt|	overview and quick reference (this file)
-|helphelp.txt|	about using the help files
-|index.txt|	alphabetical index of all commands
-|tips.txt|	various tips on using Vim
-|message.txt|	(error) messages and explanations
-|develop.txt|	development of Nvim
-|debug.txt|	debugging Vim itself
-|uganda.txt|	Vim distribution conditions and what to do with your money
+					*reference_toc* *doc-file-list* *Q_ct*
+Meta ~
+|nvim|			Transitioning from Vim
+|vim-differences|	Nvim compared to Vim
+|user-manual|		User manual: How to accomplish editing tasks.
+|quickref|		Overview of common commands
+|tutor|			30-minute interactive course for beginners
+|copying|		About copyrights
+|iccf|			Helping poor children in Uganda
+|sponsor|		Sponsor Vim development, become a registered Vim user
+|www|			Vim on the World Wide Web
+|bugs|			Where to send bug reports
+|support|		Supported platforms
+
+General ~
+|intro.txt|		Introduction to Vim; notation used in help files
```

Comment:
Nitpick: Would it be possible to homogenize (drop `.txt` everywhere)?

Reply:
I did that where possible. Some of these don't have non-txt tags. Not sure
"intro" deserves its own tag.

Can revisit this later. Much of these will be reworked anyway.